ast2vec/repo2/base.py

```diff
@@ -495,21 +496,54 @@ def result_to_model_kwargs(self, result, url_or_path: str) -> dict:
         raise NotImplementedError
 
 
-def ensure_bblfsh_is_running_noexc():
+def resolve_bblfsh_timeout(bblfsh_timeout):
+    if bblfsh_timeout is not None:
+        return bblfsh_timeout
+    log = logging.getLogger("bblfsh")
+    if "BBLFSH_TIMEOUT" in os.environ:
+        try:
+            timeout = int(os.environ["BBLFSH_TIMEOUT"])
+            log.info("Get bblfsh timeout from BBLFSH_TIMEOUT environment variable: %s sec",
+                     timeout)
+            return timeout
+        except ValueError as e:
+            log.warning(("You provide wrong value: {} in BBLFSH_TIMEOUT environment variable. "
```

Comment:
Please use the built-in logging syntax with %s
log.warning("You provide**d** wrong value: %s", env_timeout)